reqs/test.pip

```diff
@@ -1,54 +1,54 @@
-boto3==1.14.8
+boto3==1.34.56
 configparser
 h5py==2.10.0; platform_machine != "arm64" and python_version < "3.8"
-h5py==3.8.0; platform_machine != "arm64" and python_version >= "3.8"
+h5py==3.10.0; platform_machine != "arm64" and python_version >= "3.8"
 future
 numpy>1.18.4; platform_machine != "arm64"
-numpy==1.24.2; platform_machine == "arm64"
+numpy==1.26.4; platform_machine == "arm64"
 libsvm; python_version < '3.11'
-olefile==0.44
+olefile==0.47
 pandas
-parameterized==0.8.1
+parameterized==0.9.0
 pillow
-pytest==7.1.2
+pytest==8.0.2
 pytest-cov
 pytest-sugar
 
 scikit-learn==0.19.2; python_version < '3.8'
-scikit-learn==1.1.2; python_version >= '3.8'
+scikit-learn==1.4.0; python_version >= '3.8'
 
 scipy>1.4 ; python_version < '3.8'
-scipy==1.8.1; python_version >= '3.8' and python_version < '3.11'
-scipy==1.9.2; python_version == '3.11'
+scipy==1.8.1; python_version >= '3.8' and python_version < '3.12'
+scipy==1.12.0; python_version >= '3.11'
 
 six
 sympy > 1.6
-gast==0.4.0
-torch==2.1.0
-torchaudio==2.1.0
-torchvision==0.16.0
-xgboost==1.4.2; platform_machine != "arm64"
+gast==0.5.4
+torch==2.2.1
+torchaudio==2.2.1
+torchvision==0.17.1
+xgboost==2.0.3; platform_machine != "arm64"
 mock
 wrapt
 tqdm
 pytest-timeout
 
 # TensorFlow (x86) related package
-tensorflow==2.12.0; platform_machine != "arm64"
-tensorflow-estimator==2.12.0; platform_machine != "arm64"
-keras==2.12.0; platform_machine != "arm64"
+tensorflow==2.16.0; platform_machine != "arm64"
```

Comment:
This version isn't even on PyPI yet: https://pypi.org/project/tensorflow/#history

Reply:
yes, I know, only I prepared it

Reply:
tensorflow 2.16.1 is out